source/gui/__init__.py

```diff
@@ -344,6 +348,10 @@ def __init__(self, frame):
 		# Translators: The label for the menu item to open Mouse Settings dialog.
 		item = menu_preferences.Append(wx.ID_ANY, _("&Mouse settings..."),_("Change reporting of mouse shape and object under mouse"))

```

Comment:
There need to be two separate translator strings here for scons tests to pass.
Something like:
```
			item = menu_preferences.Append(wx.ID_ANY, 
				# Translators: The label for the menu item to open the Touch Interaction dialog.
				_("&Touch interaction..."),
				# Translators: The description for the menu item to open the Touch Interaction dialog.
				_("Change how NVDA interacts with the touchscreen")
			)
```